nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/S3TestColdStorage.java

```diff
@@ -66,6 +66,25 @@ public void shouldBeingRetrieved() {
         assertEquals(Boolean.TRUE, s3TestHelper.isBlobContentBeingRetrieved(documentModel));
     }
 
+    @Test
+    public void shouldRestore() {
+        DocumentModel documentModel = createFileDocument(DEFAULT_DOC_NAME, true);
+
+        // move the blob to cold storage
+        documentModel = moveContentToColdStorage(session, documentModel.getRef());
+        session.saveDocument(documentModel);
+        // undo move from the cold storage
+        documentModel = ColdStorageHelper.restoreContentFromColdStorage(session, documentModel.getRef());
+        session.saveDocument(documentModel);
+        transactionalFeature.nextTransaction();
+        documentModel.refresh();
+
+        assertEquals(Boolean.TRUE,
+                documentModel.getPropertyValue(ColdStorageHelper.COLD_STORAGE_TO_BE_RESTORED_PROPERTY));
+
+        assertEquals(Boolean.TRUE, s3TestHelper.isBlobContentBeingRetrieved(documentModel));
```

Comment:
Abdoul why we are overriding this method. the behaviour should be the same as the blob is still in `Standard` storage

Reply:
The mock has been done, the class storage is no longer Standard.